cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/EnvActionImpl/Binder/help.jelly

```diff
@@ -28,7 +28,6 @@ node {
     </p>
     <ul>
         <li><code>EXECUTOR_NUMBER</code></li>
-        <li><code>NODE_NAME</code></li>
         <li><code>NODE_LABELS</code></li>
```

Comment:
Why don't we expose NODE_LABELS? they are important for decision making on cross-platform builds and specific environments, which are being commonly via labels.